.rubocop_todo.yml

```diff
@@ -48,16 +48,6 @@ Minitest/AssertRaisesCompoundBody:
     - 'test/new_relic/agent/new_relic_service_test.rb'
     - 'test/new_relic/cli/commands/deployments_test.rb'
 
-# Offense count: 9
-# This cop supports safe autocorrection (--autocorrect).
-Minitest/AssertRespondTo:
```

Comment:
I thought this was a nice one to add since we're already using `assert_respond_to` more in our codebase than not